astropy/io/misc/asdf/schemas/astropy.org/astropy/time/timedelta-1.0.0.yaml

```diff
@@ -0,0 +1,31 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://astropy.org/schemas/astropy/time/time-1.1.0"
+title: Represents an instance of TimeDelta from astropy
+description: |
+  Represents the time difference between two times.
+
+type: object
+properties:
+    jd1:
+      type: number
+      description: |
+        Value representing first 64 bits of precision
+    jd2:
+      type: number
+      description: |
+        Value representing second 64 bits of precision
+    format:
+      type: string
+      description: |
+        Format of time value representation.
+      enum: [sec, jd, datetime]
```

Comment:
It is possible to register new formats i.e http://docs.astropy.org/en/stable/time/index.html#writing-a-custom-format and https://github.com/sunpy/sunpy/blob/master/sunpy/time/utime.py

Reply:
Sadly, I guess the easiest thing to do is to get rid of the `enum`.